src/com/epam/javacourse/homework4/HousePlant.java

```diff
@@ -0,0 +1,49 @@
+package com.epam.javacourse.homework4;
+
+public class HousePlant implements Plant {
+    String name;
+    String type;
+    String nativeRegion;
+
+    @Override
+    public void createPlant(String name, String type, String nativeRegion) {
+        this.name = name;
+        this.type = type;
+        this.nativeRegion = nativeRegion;
+        System.out.println("House plant is created.");
+    }
+
+    @Override
+    public void updatePlant(String name, String type, String nativeRegion) {
```

Comment:
`createPlant` and `updatePlant` are completely the same except for console output. We must avoid code duplication.